doc-source/scorekeep-workerthreads.md

```diff
@@ -28,15 +28,16 @@ import [com\.amazonaws\.xray\.entities\.Subsegment](https://docs.aws.amazon.com/
       Entity segment = recorder.getTraceEntity();
       Thread comm = new Thread() {
```

Comment:
This works but just to echo the question from the other thread. Many java users are likely going to be working with futures that return operation values. It would be worth having an example specifically for that.

Reply:
Thanks for the callout, I'll add some additional example code to use the SegmentContextExecutors as I mentioned in https://github.com/aws/aws-xray-sdk-java/issues/132